project-base/app/ecs-skip-rules.php

```diff
@@ -25,93 +25,45 @@
         __DIR__ . '/tests/App/Functional/EntityExtension/Model/*',
     ],
     FunctionLengthSniff::class => [
-        __DIR__ . '/src/Migrations/Version20190801103940.php',
```

Comment:
the `ecs` tool is now reporting the useless skips or was this a manual work? :open_mouth:

Reply:
it was manual work... sadly ecs does not report useless skips